### PR TITLE
Feature/add actor item

### PR DESCRIPTION
**Pull request by NadaFeteiha**

### CODE REVIEW

**Comment by NadaFeteiha** [comment]
@omarezzdeen @Karrar-Mohammed  In this pull request, you will find actorItem that can display actors' names. 
media that adapts its height to different screen sizes.
Grid of actors that can be used as the cast in movie details or at home for celebrities.

File: app/src/main/java/com/karrar/movieapp/ui/main/StartActivity.kt [hunk 1]
```diff
@@ -22,22 +27,20 @@ class StartActivity : ComponentActivity() {
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
                 ) {
-                    Greeting("Android")
+                    ActorItem(
+                        modifier = Modifier.padding(8.dp),
+                        actorName = "Test test",
+                        actorImageUrl = "https://image.tmdb.org/t/p/w500/8V1XPEDLtJoxOWlE6gYzn2sQaMp.jpg",
+                        onClick = { }
```
Comment (Karrar-Mohammed) [inline app/src/main/java/com/karrar/movieapp/ui/main/StartActivity.kt:34]: i think you are calling the actor item here just for testing if that the case don't forget to remove them before committing your work

**Review by Karrar-Mohammed — APPROVED** [review]

File: app/src/main/java/com/karrar/movieapp/ui/composable/MediaItem.kt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    onClick: () -> Unit
+) {
+    Image(
+        painter = rememberAsyncImagePainter(model = mediaImageUrl),
+        contentDescription = null,
+        contentScale = ContentScale.Crop,
+        modifier = modifier
+            .wrapContentSize()
+            .height(imageHeight.dp)
+            .width((imageHeight + 50).dp)
```
Comment (omarezzdeen) [inline app/src/main/java/com/karrar/movieapp/ui/composable/MediaItem.kt:27]: why have height and add 50?

**Review by omarezzdeen — APPROVED** [review]

File: app/src/main/java/com/karrar/movieapp/ui/composable/MediaItem.kt (hunk 2 above)
Comment (NadaFeteiha) [reply app/src/main/java/com/karrar/movieapp/ui/composable/MediaItem.kt:27]: the relation that I found between diffrent these two  (128*176), (99*143)

Screenshot 2023-03-15 at 4 41 53 PM

Screenshot 2023-03-15 at 4 41 41 PM

**Pull request merged**